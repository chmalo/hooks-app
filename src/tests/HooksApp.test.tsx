import { describe, expect, test } from "vitest"
import { render } from "@testing-library/react"
import HooksApp from "../HooksApp"

describe("Testing <HooksApp/>", () => {
  test("it should show correctly <App/> component", () => {
    const { container } = render(<HooksApp />)
    expect(container).toMatchSnapshot()
  })
})
