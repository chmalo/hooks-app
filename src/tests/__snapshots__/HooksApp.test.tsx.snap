// Vitest Snapshot v1

exports[`Testing <HooksApp/> > it should show correctly <App/> component 1`] = `
<div>
  <div>
    <h1>
      Hola mundo
    </h1>
  </div>
</div>
`;
